crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -47,6 +47,7 @@ uuid.workspace = true
 # TODO: put these behind feature flag to prevent leakage
 # Used for dummy data
 bcs.workspace = true
+move-transactional-test-runner.workspace = true
```

Comment:
There's a very hopeful "cleanup test only deps" TODO below -- is that a thing that's happening soon (by which I mean, "next")? Or at some indeterminate point in the future, because this change is evidence that it needs to happen sooner rather than later.

Reply:
Yes, very very soon indeed. Been doing general cleanup PRs this week :)

Reply:
[This](https://github.com/MystenLabs/sui/pull/14784) PR starts the cleanups. More to come.